setup.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import io
+
+from setuptools import setup
+
+# README into long description
+with io.open('README.rst', encoding='utf-8') as readme_file:
```

Comment:
Are we planning to support installing this theme for use in Py2 docs builds? If not, we can just use the open() builtin directly here, rather than going through the `io` module.